src/index.html

```diff
@@ -160,35 +158,190 @@
             <ul class="p-newjobs__list__li__salary">
               <li class="p-newjobs__list__li__salary__parts">
                 <p class="p-newjobs__list__li__salary__parts__title">給与</p>
-                <img class="p-newjobs__list__li__salary__parts__text" src="./img/picture/newjobs-text01.png" alt="月給 148,000円～">
+                <p class="p-newjobs__list__li__salary__parts__text">月給 148,000円～</p>
               </li>
               <li class="p-newjobs__list__li__salary__parts">
                 <p class="p-newjobs__list__li__salary__parts__title">所在地</p>
-                <img class="p-newjobs__list__li__salary__parts__text" src="./img/picture/newjobs-text02.png" alt="三重県伊勢市">
+                <p class="p-newjobs__list__li__salary__parts__text">三重県伊勢市</p>
               </li>
               <li class="p-newjobs__list__li__salary__parts">
                 <p class="p-newjobs__list__li__salary__parts__title">最寄駅</p>
-                <img class="p-newjobs__list__li__salary__parts__text" src="./img/picture/newjobs-text03.png" alt="伊勢市駅徒歩15分">
+                <p class="p-newjobs__list__li__salary__parts__text">伊勢市駅徒歩15分</p>
               </li>
             </ul>
           </li>
         </ul>
         <button class="p-newjobs__carousel-button_left c-carousel-button"></button>
         <button class="p-newjobs__carousel-button_right c-carousel-button"></button>
       </div>
-      <a class="p-newjobs__button__link" href="" ><div class="p-newjobs__button c-button">
-        新着求人一覧をもっと見る
-      </div></a>
+      <a class="p-newjobs__button__link" href="">
+        <div class="p-newjobs__button c-button">
+          新着求人一覧をもっと見る
+        </div>
+      </a>
     </div>
 
     <div class="p-contents">
       <div class="p-contents__img-dark">
-      <div class="p-contents__font-area">
-        <p class="p-contents__font">ギズジョブなら理想の職場がきっと見つかる</p>
-        <p class="p-contents__font">ギズジョブは完全無料の求職者向け求人サービスです。</p>
+        <div class="p-contents__font-area">
+          <p class="p-contents__font">ギズジョブなら理想の職場がきっと見つかる</p>
+          <p class="p-contents__font">ギズジョブは完全無料の求職者向け求人サービスです。</p>
+        </div>
       </div>
     </div>
+
+    <div class="p-features">
+
+        <div class="p-features__contents">
```

Comment:
以下のように `.l-inner` 使用した方が良いかと思います！
幅などがレスポンシブ対応されているので、後の実装が楽になるかと思います
スタイルが崩れないかはこちらで確認できていないので、動作確認はお願いしても良いですか？

他の箇所も同様ですね

```suggestion
        <div class="l-inner">
```

Reply:
先日口頭でも教えてもらった方法で現状".〇〇__contents"になっているものを".l-inner"に修正します！
※p-jobsearchに関しては必要な記述があるためp-jobsearch__contentsも残しておきます。